eduroam/android.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Connecting an Android device to the RIT eduroam network 
+layout: default
+
+
+---
+
+This is the complete instruction list for configuring Android to connect to eduroam. These instructions use the `P12` certificate.
+
+## Instructions
+
+Generate a personal certificate according to [this tutorial](./certificates).
+
+### Installing the certificates
+
+1. Open the Settings application and navigate to 'Install certificates'.  
+   > This may vary by phone manufacturer, as some phone manufacturers like to customize their settings menus. You can search for this menu option if you aren't able to navigate to it manually.
+   
+   On my Moto e5 running near-stock Android 8.1, the path is 'Network & Internet' > 'Wi-Fi' > 'Wi-Fi Preferences' > 'Advanced' > 'Install certificates'.
+2. Clicking 'Install certificates' should open up a file picker. Navigate to your user certificate, which is likely in your 'Downloads' folder, and choose that. You will need to enter the password you chose earlier for the certificate, and then enter a name for the certificate. Make sure under 'Credential use' you pick 'Wi-Fi'.  
+   ![The Android file picker, showing 'Downloads' folder and a file called '{blacked out} (RIT Student).p12'](/assets/img/eduroam-android/downloads-p12.png)
+   ![A dialog titled 'Extract certificate' with a password prompt](/assets/img/eduroam-android/cert-password.png)
+   ![A dialog titled 'Name the certificate' with a textbox for the name and a dropdown called 'Crdential use', with the options 'VPN and apps' and 'Wi-Fi'](/assets/img/eduroam-android/rit-user-cert-name-dialog.png)
+3. Do the same for the RIT root CA.  
+   > Some phones (like Samsung) won't allow you to install CA certificates without root, and if so, you may skip this step. Having the RIT root CA installed is recommended but not necessary.
+   
```

Comment:
how are these blockquotes intended to be formatted? they stand out a bit differently from the rest of the text. is that intentional?

Reply:
the intention was an aside or tip, slightly outside the regular flow of text so it doesn't blend in with everything else
the formatting is more intense and out of place than intended, i agree. i'll replace it with just a "Note:" prefix where needed, i think that's more of what I was aiming for anyways